network-grpc/proto/node.proto

```diff
@@ -49,14 +49,14 @@ message Transaction {
     bytes content = 1;
 }
 
-// Gossip message.
-message GossipMessage {
-    message NodeId {
-      bytes content = 1;
-    }
-
-    NodeId node_id = 1;
-    bytes content = 2;
+// Gossip message with information of a single network node.
+message NodeGossip {
+    // The node identifier.
+    bytes id = 1;
+    // The IP address.
+    string host = 2;
+    // Port number.
+    uint32 port = 3;
 }
```

Comment:
so actually here we want as before. A gossip message is:

* an ID; (the ID of the sender);
* a list of tuple (Id, Node) (the gossips);

The Sender always include itself in the list. This is how we get the information on the Node that contacted us.

Reply:
Is `Node` going to be an opaque blob, or should we detail some fields in protobuf? It can be specified as a multiaddr string.

Reply:
it is an opaque block. https://docs.rs/poldercast/0.2.1/poldercast/struct.Node.html
They implement `serde`. Just utilise `bincode` to encode/decode the whole `btreemap` and the `id`.

Reply:
It's back to sender ID + opaque nodes now. I did not segregate the ID field out of the node payloads as poldercast `Node` would serialize as a binary blob containing the id; no need to duplicate that information in protobuf.